### PR TITLE
Add from_coords and from_cascoords in models.Axis

### PR DESCRIPTION
**Pull request by wkentaro**
Screenshot from 2020-01-28 14-34-31

```
In [1]: robot
Out[1]: #<Kuka 0x7f4708a21fd0 0.000 0.000 0.000 / 0.0 -0.0 0.0>

In [2]: skrobot.models.Axis
Out[2]: skrobot.models.primitives.Axis

In [3]: skrobot.models.Axis.from_cascoords(robot)
Out[3]: #<Axis:axis_36de36ec_41db_11ea_b87f_e0d55eb26237 0x7f46ccc87f90 0.000 0.000 0.000 / 0.0 -0.0 0.0>

In [4]: viewer.add(skrobot.models.Axis.from_cascoords(robot))
```

### CODE REVIEW

File: skrobot/models/primitives.py [hunk 1]
```diff
@@ -22,6 +22,23 @@ def __init__(self,
             axis_length=axis_length,
         )
 
+    @classmethod
+    def from_coords(cls, coords, **kwargs):
+        assert isinstance(cascoords, model_module.Coordinates)
```
Comment (iory) [inline skrobot/models/primitives.py]: Typo cascoords -> coords

**Comment by wkentaro** [comment]
```
In [1]: c = robot.rarm.end_coords.worldcoords()    

In [2]: c                                          
Out[2]: #<Coordinates 0x7f13afcc4850 0.000 0.030 1.511 / 0.0 -1.6 -1.6>

In [3]: id(c)                                      
Out[3]: 139722530506832

In [4]: geom = skrobot.models.Axis.from_coords(c)  

In [5]: viewer.add(geom)                           

In [6]: robot.reset_manip_pose()                   
Out[6]: 
array([ 0.        ,  0.17453292,  0.        , -1.5707964 ,  0.        ,
        1.5707964 ,  0.        ,  0.        ,  0.        ,  0.        ,
        0.        ,  0.        ], dtype=float32)

In [7]: c                                          
Out[7]: #<Coordinates 0x7f13afcc4850 0.000 0.030 1.511 / 0.0 -1.6 -1.6>

In [8]: geom.parent                                
Out[8]: #<Coordinates 0x7f13afcc4850 0.000 0.030 1.511 / 0.0 -1.6 -1.6>

In [9]: robot.rarm.end_coords                      
Out[9]: #<CascadedCoords:rarm_end_coords 0x7f1384eb97d0 0.409 0.030 0.378 / 0.0 -1.6 -1.6>

In [10]: c                                         
Out[10]: #<Coordinates 0x7f13afcc4850 0.409 0.030 0.378 / -3.1 1.4 1.6>

In [11]: geom.parent                               
Out[11]: #<Coordinates 0x7f13afcc4850 0.409 0.030 0.378 / -3.1 1.4 1.6>

In [12]: geom.changed()                            
Out[12]: []

In [13]: geom                                      
Out[13]: #<Axis:axis_159a82ba_41ef_11ea_b87f_e0d55eb26237 0x7f135d4dd1d0 0.409 0.030 0.378 / 0.0 -0.0 0.0>
```

**Comment by wkentaro** [comment]
I have tested with below code:
```
In [1]: c = robot.rarm.end_coords.worldcoords()

In [2]: g = skrobot.models.Axis.from_coords(c)

In [3]: viewer.add(g)

In [4]: robot.reset_manip_pose()
Out[4]:
array([ 0.        ,  0.17453292,  0.        , -1.5707964 ,  0.        ,
        1.5707964 ,  0.        ,  0.        ,  0.        ,  0.        ,
        0.        ,  0.        ], dtype=float32)

In [5]: robot.init_pose()
Out[5]: array([0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0.], dtype=float32)

In [6]: c = robot.rarm.end_coords.copy_worldcoords()

In [7]: c
Out[7]: #<Coordinates 0x7f982da80690 0.000 0.030 1.511 / 0.0 -1.6 -1.6>

In [8]: g = skrobot.models.Axis.from_coords(c)

In [9]: viewer.add(g)

In [10]: robot.reset_manip_pose()
Out[10]:
array([ 0.        ,  0.17453292,  0.        , -1.5707964 ,  0.        ,
        1.5707964 ,  0.        ,  0.        ,  0.        ,  0.        ,
        0.        ,  0.        ], dtype=float32)
```

**Comment by wkentaro** [comment]
This PR also includes #100

**Comment by iory** [comment]
LGTM!

**Comment by iory** [comment]
close #100

**Pull request merged**